index.js

```diff
@@ -22,7 +27,7 @@
  *
  * @template {Element} X
  * @callback TestFunctionPredicate
- * @param {X} element
+ * @param {Element} element
  * @param {number|null|undefined} [index]
  * @param {Parent|null|undefined} [parent]
  * @returns {element is X}
```

Comment:
I don’t quite get why this works.
If it works, perfect. But it looks like it wouldn’t work to me 🤔

Reply:
> But it looks like it wouldn’t work to me 

For `isElement(node, element => xxx)`, the `element` is always `Element`, not `X` before assertion.

I'm not quite sure what wouldn't work to you.

Reply:
oh i get it, never mind!